rosdep/python.yaml

```diff
@@ -93,6 +100,13 @@ pyflakes3:
     stretch: [pyflakes3]
   ubuntu:
     xenial: [pyflakes3]
+pylint-pip:
```

Comment:
We already have python-pylint. If a native packaged version is available we generally use that following our guidelines: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md